### PR TITLE
[dom-mc] CPMD_4_1_CrayIntel_17_08

### PR DESCRIPTION
**Pull request by ajocksch**

### CODE REVIEW

File: easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08-pat-645-nogpu.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+sources = ['/apps/common/UES/easybuild/sources/c/CPMD/cpmd-v%s.tar.gz' %version]
+
+patches = [('CRAY-XE6-INTEL-MKL', 'CPMD/configure')]
+
+dependencies = [('fftw/3.3.4.10', EXTERNAL_MODULE)]
+
+skipsteps = ['configure']
+
+common_prebuildopts = './configure.sh PREFIX=%(installdir)s CRAY-XE6-INTEL-MKL &&'
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08-pat-645-nogpu.eb]: I don't see why you need to define this as a variable, couldn't you add this string directly to prebuildopts?

File: easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+prebuildopts = [
+                         common_prebuildopts,
+]
+#     'export omp=1 &&' + common_prebuildopts + ' sed -i \'s#cpmd\.x#cpmd_omp\.x#\' Makefile &&',
+
+files_to_copy = [(['bin/cpmd*.x'],'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/cpmd.x'],
+#    'files': ['bin/cpmd.x', 'bin/cpmd_omp.x'],
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08.eb]: do you need to keep this line?

File: easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08-pat-645-nogpu.eb (hunk 1 above)
Comment (ajocksch) [reply easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08-pat-645-nogpu.eb]: If we want to keep the OpenMP option, which might work for a next release, the variable is used twice. Thus it is efficient.

File: easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08.eb (hunk 2 above)
Comment (ajocksch) [reply easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08.eb]: Do we want to keep the OpenMP option?

**Comment by gppezzi** [comment]
retest this please

**Comment by jgphpc** [comment]
Let's try to not use `perftools*cscs` as explained in https://madra.cscs.ch/scs/doc/milestones/8

**Comment by jgphpc** [comment]
Open another PR for cpmd+perftools please

https://jenkins.cscs.ch/job/TestingEB/587/label=dom/console

**Pull request merged**